modules/extension/xsd/xsd-filter/src/main/java/org/geotools/filter/v1_0/DistanceUnits.java

```diff
@@ -35,9 +35,11 @@ private DistanceUnits(double distance, String units) {
         if (distance < 0) {
             throw new IllegalArgumentException("distance must be >= 0");
         }
-        if (units == null) {
-            throw new IllegalArgumentException("units must not be null");
-        }
+
+        // We still support old client queries without any units specified.
```

Comment:
The DistanceUnits constructor has been relaxed allowing null units.

Reply:
Please remove the commented out code, the comment could use a small update too. E.g.:

```suggestion
        // We still support old client queries without any units specified, thus allowing units to be null.
```